docs/docs/behaviors/macros.md

```diff
@@ -62,6 +78,32 @@ bindings
 There are a set of special macro controls that can be included in the `bindings` list to modify the
 way the macro is processed.
 
+### Parameters
+
+When creating a macro that takes parameter(s), there are control when the parameters passed to the macro are used
+within the macro itself. All of the controls are "one shot" and will change how the passed in parameters are used for the very next behavior in the `bindings` list of the macro.
+
+For example, to pass the first parameter from the macro into a `&kp` used in the macro, you would use:
+
+```
+bindings
+    = <&macro_param_1to1>
+    , <&kp MACRO_PLACEHOLDER>
+    ;
+```
+
+:::note
+The `MACRO_PLACEHOLDER` is defined as a convenient way to avoid just putting `0` in that location. It does _not_ control what parameters are substituted; only the macro controls do that.
```

Comment:
Might be useful to first explain that you still need to put *something* in for every binding, even if it will just get overwritten due to `&macro_param_XtoY`. Otherwise, it might be confusing why you would need to put `0` there, and therefore why you might use this instead.

Reply:
Yeah, good call. I expanded on this and pulled it out of the `note` aside it had been hiding in.